setuper/setuper.py

```diff
@@ -55,8 +57,30 @@ def setup_account(account_name, server_url, username, password):
     return iaso_client
 
 
-def create_account(server_url, username, password, additional_projects):
-    account_name = "".join(random.choices(string.ascii_lowercase, k=7))
+def validate_account_name(name: str) -> str:
+    random_name = "".join(random.choices(string.ascii_lowercase, k=7))
+    if not name:  # the user didn't pass a name as parameter
+        return random_name
```

Comment:
```suggestion
    if not name:  # the user didn't pass a name as parameter
        return "".join(random.choices(string.ascii_lowercase, k=7))
```

Possible tiny optimization: this could avoid a small useless computation.